parm/wflow/prep.yaml

```diff
@@ -50,12 +50,19 @@ task_make_orog:
         text: '{{ task_make_grid.GRID_DIR }}/make_grid_task_complete.txt'
 
 task_make_sfc_climo:
-  <<: *default_task
-  command: '&LOAD_MODULES_RUN_TASK; "make_sfc_climo" "&JOBSdir;/JREGIONAL_MAKE_SFC_CLIMO"'
-  envars:
-    <<: *default_envars
+  account: '&ACCOUNT;'
+  attrs:
+    cycledefs: at_start
+    maxtries: '2'
+  command: 'source &USHdir;/load_modules_wflow.sh {{ user.MACHINE }}; python &SCRIPTSdir;/make_sfc_climo.py
+    -c &GLOBAL_VAR_DEFNS_FP; 
+    --key-path task_make_sfc_climo'
   join: !cycstr '&LOGDIR;/{{ jobname }}_@Y@m@d@H&LOGEXT;'
-  nnodes: 2
+  nodes: "{{ task_make_sfc_climo.sfc_climo_gen.execution.batchargs.nodes }}:ppn={{ task_make_sfc_climo.sfc_climo_gen.execution.batchargs.tasks_per_node }}"
+
+  partition: '{% if platform.get("PARTITION_DEFAULT") %}&PARTITION_DEFAULT;{% else %}None{% endif %}'
+  queue: '&QUEUE_DEFAULT;'
+  walltime: 00:20:00
```

Comment:
Let's take this one from the `execution` section for `task_make_sfc_climo`, too.

Reply:
Let's go with `00:05:00`, but only in the `config_defaults.yaml` file. Here, you should use a Jinja expression to point to that value.

Reply:
Updated!